index.js

```diff
@@ -19,9 +19,13 @@ const getVersionsToDelete = async () => {
                         versions(last: ${versionLimit}){
                             edges {
                                 node {
-                                    id, 
-                                    updatedAt, 
+                                    id
                                     version
+                                    files(first: 1) {
```

Comment:
what does the `first: 1` do? does that make sense?

Reply:
Honestly, I'm not completely sure.

From files I got a list of file (all with the same date). I don't know what's in there really (my packages all had a single file, but I got a list of them anyway. Could be metadata). But they all had the same date, so I figured just to go with the first one